pages/linux/flock.md

```diff
@@ -20,6 +20,6 @@
 
 `flock {{path/to/lock.lock}} {{[-w|--timeout]}} 10 {{command}}`
 
-- Backup a bunch of files, waiting for the previous tar command to finish if it's still running elsewhere and holding the same lock file (can be used in a cronjob that runs often):
+- Backup a bunch of files, waiting for the previous `tar` command to finish if it's still running elsewhere and holding the same lock file (can be used in a `cronjob` that runs often):
```

Comment:
Maybe it's nitpicking at this point, while backticks around tar absolutely make sense of course, I'm not sure about the ones around "cronjob", it's not directly a command per se, but used as a sort of concept here.

Maybe it could be "cron job", "*cron job*", "*cron* job" or something, but backticks seem to give off a bit wrong idea here imo...

Reply:
Within the spirit of the examples listed, maybe "`cron` job"? (without quotes)

> (can be used in a `cron` job that runs often)

Reply:
Sure. Go ahead and make the edit.